code-challenges.js

```diff
@@ -17,12 +34,30 @@ var testingArray1 = [23, "Heyyyy!", 45, -9, 0, "Yo", false]
 var testingArray2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
-
-
+// create a function that is called numOnly and takes in an array that will return things % 3
+numOnly = (arr) => {
+let orgArr = arr.filter(el => typeof el === "number")
+// new array to push into with the for loop.
+let newArr = []
+ for(let i=0; i < orgArr.length; i++){
+     newArr.push(orgArr[i] % 3)
+ }
+ return newArr
+}
```

Comment:
good job! you can also add the logic of your for loop into the filter to shorten your code if you want :)